docs/04.guides/04.cookbooks/33.Hidden_gems/page.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Hidden Gems
+id: hidden_gems
+---
+## Hidden Gems ##
+
+This document explains how to declare a variables, function call with dot and bracket notation, Passing arguments via URL/form scopes as array. Explain this three with simple example below:
```

Comment:
***ToChange***
This document explains 3 features of Lucee: how to declare variables, function calls with dot and bracket notation, 
 and Passing arguments via URL/form scopes as an array. These features are explained with three simple examples below: